### ISSUE TITLE
Year Progressions Overivew - "with target" value incorrect

### ISSUE THREAD

**Issue by waldev** [thomaschampagne/elevate#1033]
When viewing the overview on the "Yearly Progressions" page, the difference "with target" for the current year is incorrect today (first day of month). Yesterday it was OK.

When you move the mouse over the graph, the "with target" updates according to the day, and appears correct, except for today (first day of month). I'm not sure if this is a "once off" or if this happens on the first day over every month.

![Capture](https://user-images.githubusercontent.com/4507145/109435206-6f2a5000-7a6d-11eb-833a-3f031bcd86cc.JPG)

**Comment by waldev** [comment]
Initially thought this may have been an issue related the month change/first day of month, but over the last few days the issues persists.
![Capture](https://user-images.githubusercontent.com/4507145/110072970-b9476480-7dd2-11eb-8d08-5d592ba3ba16.JPG)

**Comment by neuhausj** [comment]
Agreed !
The "delta with target" is not updated to the latest value when the mouse is not over the graph.
![image](https://user-images.githubusercontent.com/182152/115197206-049ea380-a0f1-11eb-962b-8038b645ade1.png)

**Comment by waldev** [comment]
Won't fix?

This is a useful metric!

**Comment by waldev** [comment]
Please fix

**Comment by waldev** [comment]
Please fix

**Comment by Ange2322** [comment]
I'm experiencing this currently. On the Chrome add on version and the desktop version as well. Is it being looking into? The Chrome version was working fine for me until fairly recently (a few weeks or a month or two ago?).

**Comment by waldev** [comment]
Having the same issue when using the Chrome extension and desktop app. Same as @Ange2322, it was working up until a short time ago, I think it came back in the desktop app after the recent update.

**Comment by Ange2322** [comment]
Is this being looked into? Surely it is very easy to fix? It's still happening on both desktop and Chrome versions. Please fix asap. Thank you.

**Comment by waldev** [comment]
Still waiting on a fix for this issue

**Comment by waldev** [comment]
Still waiting on a fix for this issue
